packages/underlay/README.md

```diff
@@ -22,3 +22,13 @@ When looking to leverage the `Underlay` base class as a type and/or for extensio
 ```
 import { Underlay } from '@spectrum-web-components/underlay';
 ```
+
+## Example
+
+```html
+<sp-underlay open></sp-underlay>
+<div class="overlay">
+    <h1>Hello, I'm an overlay!</h1>
+    <p>Enjoy your day...</p>
+</div>
+```
```

Comment:
This needs just a little bit more as the `<sp-underlay open>` will by default occlude the whole page...

```suggestion
```html
When leveraging an `<sp-underlay>` in conjunction with overlay content, place it as a sibling prior to your overlay content. 

```html
<style>
sp-underlay:not([open]) + sp-dialog {
    display: none;
}
sp-underlay + sp-dialog {
    position: fixed;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    z-index: 1;
    background: var(--spectrum-dialog-confirm-background-color, var(--spectrum-alias-background-color-default));
}
</style>
<sp-button
    onclick="
        console.log(this.nextElementSibling);
        this.nextElementSibling.open = true;
    "
>Open dialog with underlay element</sp-button>
<sp-underlay></sp-underlay>
<sp-dialog size="small">
    <h1 slot="heading">Hello, I'm an overlay!</h1>
    <p>Enjoy your day...</p>
    <sp-button
        slot="button"
        onclick="
            this.parentElement.previousElementSibling.open = false;
        "
    >Close</sp-button>
</sp-dialog>
```
```

Reply:
Great suggestion! updated: https://github.com/adobe/spectrum-web-components/pull/1681/commits/aeadd1bb4d8f090183903c980fcf576098a0b800